plugins/modules/digital_ocean_droplet.py

```diff
@@ -344,7 +344,7 @@ def create(self, state):
             droplet_id = droplet_data.get("id", None)
             if droplet_id is not None:
                 if self.wait:
-                    if state == "active":
+                    if state == "present" or state == "active":
```

Comment:
What is the difference between "present" and "active" here? Were we previously assuming (incorrectly) that "present" means "the droplet is in the account but not necessarily powered on"?

Reply:
Yep; we actually support four (4) states: `present` (exists), `active` (exists and running), `inactive` (exists and not running) and `absent` (doesn't exist).